### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/convertToObject.js [hunk 1]
```diff
@@ -3,16 +3,29 @@
...
+    .map(item => {
+      cssStyles[item[0].trim()] = item[1].trim();
+    });
 
+  return cssStyles;
+}
 module.exports = convertToObject;
+// for(var key in input){
+//   output[key.replace(/\s+/,'_')] = input[key];
+// }
```
Comment (Alej4ndro1) [inline src/convertToObject.js:31]: You need to delete comments

**Review by Alej4ndro1 — APPROVED** [review]
Good but delete comments

File: src/convertToObject.js [hunk 2]
```diff
@@ -3,16 +3,26 @@
...
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const cssStyles = {};
+
+  sourceString.split(';')
+    .filter(item => item.trim())
+    .map(item => item.split(':'))
+    .map(item => {
```
Comment (AndriiRut) [inline src/convertToObject.js:22]: item is the synonym of element, and makes the code less readable. use such names as style, properties, values, string and etc)

checklist(NAMING)

**Review by AndriiRut — CHANGES_REQUESTED** [review]

**Review by AndriiRut — APPROVED** [review]
🚀

**Pull request closed without merging**